proto/p4/p4runtime.proto

```diff
@@ -50,88 +78,56 @@ service P4Runtime {
   // Reads 'CounterEntry's.
   rpc CounterRead(CounterReadRequest) returns (stream CounterReadResponse) {
   }
```

Comment:
@aghaffarkhah no I'm using them, we can remove them at the same time as the rest when we cleanup